### ISSUE TITLE
Add rendering for vending=bottle_return

### ISSUE THREAD

**Issue by richlv** [gravitystorm/openstreetmap-carto#4580]
It would be great to render vending=bottle_return both on nodes and closed ways.
https://wiki.openstreetmap.org/wiki/Tag:vending%3Dbottle_return

Deposit return places exist in many countries, but the mapping of these is rather poor - see Europe+ below.
This might be related to these locations not being rendered, and having them show up would likely increase mapping significantly.
![image](https://user-images.githubusercontent.com/112888/176664149-debec074-2dca-4ac5-924c-56e5c99ac37f.png)

Several possibilities for the icon, one being:

![image](https://user-images.githubusercontent.com/112888/176664922-79d9b857-c6c2-40e7-a46a-3423078c1d36.png)
(taken from https://eng.lsm.lv/article/society/environment/latvias-deposit-system-101-what-where-and-how.a441369/ )

**Comment by imagico** [comment]
Thanks for the suggestion.  As you already indicate the tag in question is used so far in very low volume (420 uses, almost all on nodes) and only in very few countries.  And practice and the legal framework for recycling bottles differs a lot world wide (and so does the prevalence of different methods of collection and types of containers for beverages - cans, bottles etc.).  See in particular how much the usage map of the tag correlates with

https://commons.wikimedia.org/wiki/File:Container-deposit_legislation_in_Europe_SVG_map.svg

Most of the existing use of the tag is for indoor infrastructure, often for machines within shops only accessible to customers during opening hours of the shop (because that is the business model legislation in those countries induces).  The tag seems to be combined both with `amenity=vending_machine` and `amenity=recycling` and there does not appear to be consensus on the semantic delineation between those two.  There is also some (small volume) use of `recycling_type=machine|vending_machine|station|reverse_vending_machine|deposit_machine` and it is likely that there is a significant volume of such machines (and also more low tech infrastructure like https://de.wikipedia.org/wiki/Pfandring) that is tagged `amenity=recycling` without `vending=bottle_return`.

Long story short - this is quite clearly a field were mapping and tagging in OSM is still in development and no clear consensus exists among mappers how to map such infrastructure in a way that has a clear meaning across different cultures and legislations.  Hence i would suggest not to render this at this time.

Side note: What would be worth considering is if we can render `amenity=recycling` at the highest zoom levels in a more differentiated fashion indicating the type of object/material that can be deposited based on `recycling:*=yes|no` - use of those tags exists in numbers up to more than 100k: https://taginfo.openstreetmap.org/tags/amenity=recycling#combinations

**Comment by richlv** [comment]
And now we have people tagging these as recycling centres, because.
While I'm not doing that myself (yet?) I cannot blame them that much.
Deposit return places are of more interest (and more permanent) than plain recycling containers, which actually get rendered.
The mess in tagging will only increase.

**Comment by DnaX** [comment]
Why this issue it's closed? For Example in Romania it's in place since 1 year the refund system for PET/glass bottles and cans. Are machines outside the supermarkets, so an actual POI. I also added a preset into NSI: https://nsi.guide/?id=returo-33f2a6

This tag should be rendered with the same color of recycling:* and some kind of icon (or point).

Now we are at 1,523 usage.
